### PR TITLE
feat: Vue Apollo docs in French 🇫🇷

### LINKED ISSUE #919: [Translation] Vue Apollo in French 🇫🇷
**Is your feature request related to a problem? Please describe.**

The documentation is currently only in English and Chinese.

**Describe the solution you'd like**

A translation of the documentation in French.

Hey folks! Thanks for the awesome package. I'm willing to contribute to the docs by adding the complete French translation to help the library reach even more people.

### PR DESCRIPTION
**Pull request by sarahdayan**
This PR translates the Vue Apollo documentation in French.

Since there were no guidelines on how to PR translations, I made a single big PR, taking the example of the pending Spanish translation PR.

I did my best to follow the [vuejs-fr terminology](https://github.com/vuejs-fr/vuejs.org/issues/4).

fix #919

### CODE REVIEW

File: packages/docs/src/fr/api/apollo-mutation.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- `refetchQueries`: Consultez le guide sur [comment re-requêter après une mutation](https://www.apollographql.com/docs/react/data/mutations/#options)
+- `clientId`: l'identifiant du client Apollo utilisé par la requête (défini dans l'option `clients` d'ApolloProvider)
+- `tag`: le nom de la balise HTML (par défaut: `div`); si `undefined`, le composant n'a pas de rendu (le contenu ne sera pas englobé dans une balise)
+- `context`: Consultez [l'option `context` d'Apollo](https://www.apollographql.com/docs/react/data/mutations/#options)
+
+## Les props de slots avec portée
+
+- `mutate(options = undefined)`: une fonction pour appeler une mutation. Vous pouvez écraser les options de mutation (par exemple : `mutate({ variables: { foo: 'bar } })`)
+- `loading`: un booléen qui indique que la requête est en cours
+- `error`: un erreur évntuelle lors de la dernière mutation
```
Comment (netgusto) [inline packages/docs/src/fr/api/apollo-mutation.md]:
```suggestion
- `error`: une erreur éventuelle lors de la dernière mutation
```

File: packages/docs/src/fr/api/apollo-mutation.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- `gqlError`: la première erreur GraphQL évntuelle
```
Comment (netgusto) [inline packages/docs/src/fr/api/apollo-mutation.md]:
```suggestion
- `gqlError`: la première erreur GraphQL éventuelle
```

File: packages/docs/src/fr/api/apollo-subscribe-to-more.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      return newResult
+    },
+  },
+}
+</script>
+```
+
+## Props
+
+- `document`: un document GraphQL qui contient la souscription ou une fonction qui reçoit le gabrit `gql` en argument et doit retourner le document transformé.
```
Comment (netgusto) [inline packages/docs/src/fr/api/apollo-subscribe-to-more.md]:
```suggestion
- `document`: un document GraphQL qui contient la souscription ou une fonction qui reçoit le gabarit `gql` en argument et doit retourner le document transformé.
```

**Comment by Akryum** [comment]
That's an amazing work! Maybe you could rebase and update from latest dev so I can merge it? Thanks!

**Pull request merged**